back-end/objects/RModelWrapper.py

```diff
@@ -52,6 +52,41 @@ def __init__(
         else:
             print("Checkpoint file not found: {}".format(net_path))
 
+        # TODO: Should remove this in the future. Currently added for passing the e2e test.
+        # Also stop passing app to this class.
+        self.upload_model_4_e2e_test(app)
+
+    def upload_model_4_e2e_test(self, app):
```

Comment:
This is a hacky way to make things work. Let's add a todo here when model upload works end-to-end.

```suggestion
    # TODO(Chonghan): upload a model at the start of e2e tests, and remove this afterwards
    def upload_model_4_e2e_test(self, app):
```